src/core/crypto.js

```diff
@@ -2013,7 +2014,15 @@ var CipherTransformFactory = (function CipherTransformFactoryClosure() {
     this.encryptionKey = encryptionKey;
 
     if (algorithm >= 4) {
-      this.cf = dict.get('CF');
+      var cf = dict.get('CF');
+      if (isDict(cf)) {
```

Comment:
I'm a bit confused about how this check indicates that the `CF` dictionary is encrypted, as the comment below states. We're just checking if it's a dictionary, right, not that it's encrypted, or am I missing something here?